packages/react-core/src/components/TreeView/TreeView.tsx

```diff
@@ -23,10 +23,8 @@ export interface TreeViewDataItem {
   hasBadge?: boolean;
   /** Additional properties of the tree view item badge */
   badgeProps?: any;
-  /** Action of a tree view item, nested inside a button */
+  /** Action of a tree view item, can be a Button or Dropdown */
   action?: React.ReactNode;
-  /** Additional properties of the tree view item action button */
-  actionProps?: any;
```

Comment:
I know TreeView is relatively new, but isn't this a breaking change?

Reply:
Yeah it is. It's a beta component though, I thought breaking changes were ok / covered by the disclaimer?

Reply:
@redallen should I make a note/label somewhere indicating the breaking changes, or will that screw up semantic release versioning since it's beta?

Reply:
Just make note of it in the OP of this PR for the release notes.

Reply:
Ok, didn't realize this was still beta